lib/client/wallet.js

```diff
@@ -285,7 +301,7 @@ class WalletClient extends Client {
 
   /**
    * Get wallet blocks.
-   * @param {Number} height
+   * @param {Number} id
    * @returns {Promise}
    */
 
```

Comment:
Are you sure this is actually an `id` and not a `height`? What is a block id?

Reply:
Nevermind, it is an id
https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/http.js#L554